src/NuGet.Core/NuGet.ProjectManagement/Projects/FolderNuGetProject.cs

```diff
@@ -27,8 +27,8 @@ public class FolderNuGetProject : NuGetProject
         /// </summary>
         public PackageSaveModes PackageSaveMode { get; set; }
 
-        public FolderNuGetProject(string root)
-            : this(root, new PackagePathResolver(root))
+        public FolderNuGetProject(string root, bool excludeVersion = false)
```

Comment:
It's a good idea to avoid optional parameters on public APIs. Just make a new constructor that takes excludeVersion.